schemas/com.delighted/survey_response_created/jsonschema/1-0-0

```diff
@@ -0,0 +1,163 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Delighted survey response created",
+    "self": {
+        "vendor": "com.delighted",
+        "name": "survey_response_created",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "event_type": {
+            "type": "string",
+            "enum": ["survey_response.created"],
+            "description": "This describes what kind of event triggered this webhook. It is composed of an object type and an action, separated by a period"
+        },
+        "event_id": {
+            "type": "string",
+            "maxLength": 64,
+            "description": "The event_id can be used to deduplicate notifications that may be delivered more than once"
+        },
+        "event_data": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "maxLength": 20,
+                    "description": "Response id"
+                },
+                "person": {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "string",
+                            "description": "Responder id",
+                            "maxLength": 25
+                        },
+                        "email": {
+                            "type": ["string", "null"],
+                            "maxLength": 255,
+                            "description": "Responder email address"
+                        },
+                        "name": {
+                            "type": ["string", "null"],
+                            "maxLength": 255,
+                            "description": "Responder name"
+                        },
+                        "created_at": {
+                            "type": "integer",
```

Comment:
Using integers for all the timestamps is going to bleed complexity downstream into all consumers of these events. How difficult would it be to convert these in the adapter?

Reply:
I've thought about the best way to do this a bit. A large number of webhooks just give epoch timestamps rather than nicely formatted ISO datetimes. It's possible to do these quite easily in the adapter but it does mean that all webhooks with epochs require adapters.